plugin/src/main/java/net/dzikoysk/funnyguilds/feature/hooks/holographicdisplays/HologramsUtils.java

```diff
@@ -0,0 +1,20 @@
+package net.dzikoysk.funnyguilds.feature.hooks.holographicdisplays;
+
+import net.dzikoysk.funnyguilds.FunnyGuilds;
+import net.dzikoysk.funnyguilds.config.sections.HologramConfiguration;
+import net.dzikoysk.funnyguilds.guild.Guild;
+import org.bukkit.Location;
+import panda.std.Option;
+
+public class HologramsUtils {
+
+    private HologramsUtils() {}
+
+    public static Option<Location> calculateLocation(Guild guild) {
+        HologramConfiguration hologram = FunnyGuilds.getInstance().getPluginConfiguration().heartConfig.hologram;
```

Comment:
1. zamiast tutaj bez sensu wyciągać, to lepiej już zrobić to jako zmienną w lambdzie w map poniżej
2. to tak właściwie nie powinien nawet być utils, bo bazuje na configu, raczej to powinno być w jakimś managerze hologramów czy coś